packages/flet/lib/src/controls/list_view.dart

```diff
@@ -79,14 +79,31 @@ class _ListViewControlState extends State<ListViewControl> {
                 shrinkWrap: shrinkWrap,
                 padding: padding,
                 semanticChildCount: semanticChildCount,
-                itemExtent: itemExtent,
-                prototypeItem: prototypeItem,
-                children: controls
-                    .map((item) => ControlWidget(
-                          key: ValueKey(item.getKey("key")?.value ?? item.id),
-                          control: item,
-                        ))
-                    .toList(),
+                itemExtent: spacing > 0 ? null : itemExtent,
+                prototypeItem: spacing > 0 ? null : prototypeItem,
```

Comment:
Can this be mentioned in the python documentation of these props?
Kind of:
```
Note:
    Has effect only when spacing is ...
```

This makes me actually question: spacing is usually a value greater than or equal to 0; so in what case are these props set? when spacing is = 0 only?

Reply:
If spacing is not zero setting item extent or prototype does not have any sense as spacing adds width/height to item extent.

I would think we can try wrapping prototype item into row/column and add a spacer, but then what would be the hight/width of the last element?..

Reply:
I don't think item extent and prototype helps a lot as it's "non-build-on-demand" mode which is not default and should be used for building relatively small lists. For large lists built on-demand we use `ListView.separated()` and `ListView.builder()`.